cmd/cluster_test.go

```diff
@@ -17,16 +17,15 @@ package cmd_test
 
 import (
 	"fmt"
-	"net/http"
-	"os"
-	"os/exec"
-
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 	"github.com/onsi/gomega/gbytes"
 	"github.com/onsi/gomega/gexec"
 	"github.com/onsi/gomega/ghttp"
 	openapi "github.com/yugabyte/yugabytedb-managed-go-client-internal"
+	"net/http"
```

Comment:
Not sure which formatter you are using but normally go internal lib should be first.